FND/Relations/Relations.rdf

```diff
@@ -63,19 +63,19 @@ the FIBO FND RFC, was revised in advance of the September 2013 New Brunswick, NJ
 		<skos:changeNote>The https://spec.edmcouncil.org/fibo/ontology/FND/20190501/Relations/Relations.rdf version of this ontology was modified to rename (migrate) the hasDefinition property to isDefinedIn.</skos:changeNote>
 		<skos:changeNote>The https://spec.edmcouncil.org/fibo/ontology/FND/20190701/Relations/Relations.rdf version of this ontology was modified to eliminate deprecated elements.</skos:changeNote>
 		<skos:changeNote>The https://spec.edmcouncil.org/fibo/ontology/FND/20190901/Relations/Relations.rdf version of this ontology was modified to eliminate duplication with concepts in LCC and remove the unused hasDispositionDate property, whose semantics were unclear at best.</skos:changeNote>
+		<skos:changeNote>The https://spec.edmcouncil.org/fibo/ontology/FND/20200301/Relations/Relations.rdf version of this ontology was modified to move hasAcquisitionDate to Financial Dates to improve usability and simplify reasoning, and to eliminate circular definitions.</skos:changeNote>
 		<fibo-fnd-utl-av:hasMaturityLevel rdf:resource="&fibo-fnd-utl-av;Release"/>
 	</owl:Ontology>
 	
 	<owl:Class rdf:about="&fibo-fnd-rel-rel;Reference">
 		<rdfs:label>reference</rdfs:label>
-		<skos:definition>a concept that refers to (or stands in for) another concept</skos:definition>
-		<fibo-fnd-utl-av:adaptedFrom rdf:datatype="&xsd;anyURI">http://grammar.about.com/od/rs/g/referenceterm.htm</fibo-fnd-utl-av:adaptedFrom>
+		<skos:definition>idea, notion, condition, or assumption that determines how something may be approached, perceived, or understood</skos:definition>
```

Comment:
The definition of reference was, like ownership, identified as being circular by the hygiene test.  

We've been asked many times to simplify the class hierarchy, have fewer elements at the top when using tools like TopBraid and Protge, or even in UML, so I am reticent to remove this.  The concepts comes from linguistics.  I'll attempt to clarify the definition and add an example, but I do not want to remove this because of statements users have made.  In fact, we should raise an issue to further collapse the hierarchy in cases where that is possible, though resource constraints make that a difficult sell.